tss-esapi-sys/src/bindings/x86_64-unknown-linux-gnu.rs

```diff
@@ -3498,13 +3474,13 @@ impl Default for TPMS_AUTH_RESPONSE {
 }
 pub type TPMI_AES_KEY_BITS = TPM2_KEY_BITS;
 pub type TPMI_SM4_KEY_BITS = TPM2_KEY_BITS;
-pub type TPMI_CAMELLIA_KEY_BITS = TPM2_KEY_BITS;
```

Comment:
This kind of name change is a breaking change because these constants and names are exposed through the crate.